iron-a11y-keys-behavior.html

```diff
@@ -208,6 +173,7 @@
 
         if (keyName in MODIFIER_KEYS) {
           parsedKeyCombo[MODIFIER_KEYS[keyName]] = true;
+          parsedKeyCombo.checkModifiers = true;
```

Comment:
Since a `keyCombo` is part of the public implementation of this behavior, I would like to suggest that this property be called `hasModifiers`. As a phrase, `checkModifiers` leaks implementation details that the user isn't aware of and shouldn't need to be.